OCAT/client/resources/js/components/shared/services/assessment.service.js

```diff
@@ -1,15 +1,13 @@
 import axios from "axios";
-
 export class AssessmentService {
-    static async submit (assessment) {
-        try {
-            //Choose the correct method, url, and data to send in a request to the express OCAT/server/routes
-            await axios({});
+  static async submit(assessment) {
 
-            return;
-        }
-        catch (err) {
-            throw new Error(`${err.response.statusText} - ${err.response.data.message}`);
-        }
+    try {
+      console.log(assessment)
+      await axios.post('http://localhost:3000/assessment/submit', assessment);
```

Comment:
```suggestion
      await axios.post('${config.api.host}/assessment/submit', assessment);
```
this is another good use case for the config.json.  No reason to supply your host over and over again in the code